### DISCUSSION TITLE
NEW: How do I create a Tor website?

### DISCUSSION THREAD

**Discussion by doctorstrange2001** [community/community#32133] [Other Feature Feedback, Questions, & Ideas]
Hi, I've asked for help with this before, but I was advised to ask questions. This revolves around creating a website via Github, for Onionshare and Tor:
1. How do I choose the design for my website?
2. How make I a official name for my website?
3. How do I paste the logo and texts on my website?
4. How do I create multiple website pages?
5. How do I make sure that one page links to another page?
6. How do I save everything in a file on my laptop or computer?

You can email me to:
[Redacted due to PII]

**Comment by AllisonGarner** [comment]
I've found this post on Tor's official community forum. This might help you.

https://community.torproject.org/onion-services/setup/

**Comment by queenofcorgis** [comment]
Hi @doctorstrange2001,

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by Kirtishukla2004** [comment]
https://www.makeuseof.com/tag/create-hidden-service-tor-site-set-anonymous-website-server/#:~:text=How%20to%20Create%20a%20Hidden%20Service%20Tor%20Site,Server.%203%20Configure%20The%20Hidden%20Service.%20See%20More.

go on the link hope it  will help  you
